FLT/NumberField/Completion/Infinite.lean

```diff
@@ -85,6 +85,8 @@ instance : Module.Free v.Completion wv.1.Completion :=
   Module.free_of_finite_type_torsion_free'
 
 variable (L v)
+
+open Classical in
```

Comment:
I've always used `open scoped Classical` (to only enable the instances from the namespace, not import the declarations in it). I'm not sure if this is important.

Reply:
The same comment applies elsewhere below and in the other file with this change.

Reply:
I made these changes to fix linter warnings.

Reply:
I know and agree --- I just realised my wording was misleading: I wanted to say I prefer `open scoped Classical in` over `open Classical in`.

Reply:
Oh sure, sorry. I'll do it in a few minutes.

Reply:
Done.